account/tests/test_views.py

```diff
@@ -36,7 +37,8 @@ def test_closed(self):
             }
             response = self.client.post(reverse("account_signup"), data)
             self.assertEqual(response.status_code, 200)
-            self.assertEqual(response.template_name, "account/signup_closed.html")
+            self.assertEqual(response.template_name,
+                             "account/signup_closed.html")
```

Comment:
Fixed! The changes were because the lines exceed 79chars breaking PEP8 https://www.python.org/dev/peps/pep-0008/#id19